gazelle/python/python_test.go

```diff
@@ -152,9 +149,7 @@ func testPath(t *testing.T, name string, files []bazel.RunfileEntry) {
 
 		args := []string{"-build_file_name=BUILD,BUILD.bazel"}
 
-		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
-		t.Cleanup(cancel)
-		cmd := exec.CommandContext(ctx, gazellePath, args...)
+		cmd := exec.Command(gazellePath, args...)
```

Comment:
nit: why are we removing the 2 second timeout here? Does running `gazelle` become really expensive because of the needing to write the `embedded` file? What is roughly the impact?

Reply:
yeah, writing and extracting dozens of zip files take time. I introduced a new variable so tests can use the runfiles directly